.github/workflows/build-and-push.yml

```diff
@@ -26,13 +26,13 @@ jobs:
           # The Dockerfile file name will be inferred from the image-name parameter
           # If the image-name is chatterino2-xd, the Dockerfile it will look for is Dockerfile_chatterino2-xd
           - os: ubuntu-20.04
-            qt-version: 6.6.1
```

Comment:
Can I remove the `qt-version` options, because they don't seem to be used in that workflow file or should I leave it at updating the entries to 6.7.2?

Reply:
You can remove them, they are copy-pasted from the c2 build workflow